lib/features/auto-place/AutoPlace.js

```diff
@@ -25,6 +25,12 @@ export default function AutoPlace(eventBus, modeling) {
    */
   this.append = function(source, shape) {
 
+    // notify interested parties on auto place started
```

Comment:
We're basically firing the same event twice.

Reply:
Not Sure. If we listen to autoPlace.end, the directEditing got still activated for the newShape, so we won't be able to complete it for the source in the LabelEditingProvider

Reply:
We should complete it just _before_ the new shape was created.